lib/classes/Variables.js

```diff
@@ -889,7 +889,7 @@ class Variables {
     if (
       valueToPopulate === null ||
       typeof valueToPopulate === 'undefined' ||
-      (typeof valueToPopulate === 'object' && _.isEmpty(valueToPopulate))
+      (!_.isArray(valueToPopulate) && _.isEmpty(valueToPopulate))
```

Comment:
Let's use native`Array.isArray` (at least with recent updates we try to fallback to `lodash` only if language on its own doesn't provide an option)